hpx/util/command_line_handling.hpp

```diff
@@ -24,6 +24,21 @@
 ///////////////////////////////////////////////////////////////////////////////
 namespace hpx { namespace util
 {
+    ///////////////////////////////////////////////////////////////////////////
+    // Helper functions for retrieving command line options (with error
+    // checking)
+    HPX_EXPORT std::size_t get_num_high_priority_queues(
+        util::command_line_handling const& cfg, std::size_t num_threads);
+
+    HPX_EXPORT std::string get_affinity_domain(util::command_line_handling const& cfg);
+
+    HPX_EXPORT std::size_t get_affinity_description(
+        util::command_line_handling const& cfg, std::string& affinity_desc);
+
+    HPX_EXPORT std::size_t get_pu_offset(util::command_line_handling const& cfg);
+
+    HPX_EXPORT std::size_t get_pu_step(util::command_line_handling const& cfg);
```

Comment:
Do we have to export these functions?

Reply:
No, I don't think so. Will remove. Thanks!